config/pitest-suppressions/pitest-coding-1-suppressions.xml

```diff
@@ -156,10 +156,10 @@
   <mutation unstable="false">
     <sourceFile>VariableDeclarationUsageDistanceCheck.java</sourceFile>
     <mutatedClass>com.puppycrawl.tools.checkstyle.checks.coding.VariableDeclarationUsageDistanceCheck</mutatedClass>
-    <mutatedMethod>getFirstNodeInsideTryCatchFinallyBlocks</mutatedMethod>
-    <mutator>org.pitest.mutationtest.engine.gregor.mutators.experimental.NakedReceiverMutator</mutator>
-    <description>replaced call to com/puppycrawl/tools/checkstyle/api/DetailAST::getLastChild with receiver</description>
-    <lineContent>final DetailAST finalBlock = currentNode.getLastChild();</lineContent>
+    <mutatedMethod>calculateDistanceBetweenScopes</mutatedMethod>
```

Comment:
we need to kill this mutation.
we do not allow new technical dept.

Reply:
Done.